lib/vmpooler/api.rb

```diff
@@ -5,6 +5,8 @@ def initialize
       config_file = File.expand_path('vmpooler.yaml')
       $config = YAML.load_file(config_file)
 
+      $config[:uptime] = Time.now
```

Comment:
So this comment is me not knowing what Ruby's `Time.now` is really going to do.

When I look at Time.now, irb defaults to displaying it as a string, which corrects for DST. However, when I do math with it, it seems to know what's up. In Python, time is extremely verbose. Do you know how this will behave around DST, or would it be safer to do convert it to an integer, where we can guarantee it'll be something like:

```
$ date +%s
1423083750
$ irb
>> Time.now.to_i
=> 1423083756
>> 
$ 
```